pkg/controller/migplan/registry.go

```diff
@@ -214,3 +266,72 @@ func (r ReconcileMigPlan) ensureRegistryService(client k8sclient.Client, plan *m
 
 	return nil
 }
+
+func (r ReconcileMigPlan) getBothClients(plan *migapi.MigPlan) ([]compat.Client, error) {
```

Comment:
`getBothClients` is duplicating functionality we already have in mig-controller. Let's drop the `getBothClients` function and construct the two clients in deleteImageRegistryResources as shown below.

See 
```
# from https://github.com/konveyor/mig-controller/blob/master/pkg/apis/migration/v1alpha1/migplan_types.go#L137
plan.GetRefResources(client)
```

```
# use this in deleteImageRegistryResources()

planRefResources, err := r.GetRefResources(r.client)
if err != nil {
  return liberr.wrap(err)
}
srcClient, err := planRefResources.SrcMigCluster.GetClient(r.Client)
if err != nil {
  return liberr.wrap(err)
}
destClient, err := planRefResources.DestMigCluster.GetClient(r.Client)
if err != nil {
  return liberr.wrap(err)
}
clients := []compat.Client{srcClient, destClient}

[...]
```